main.py

```diff
@@ -51,13 +58,27 @@
         mqtt_thread: Thread = Thread(
             target=mqtt_client.run, name="ISAR MQTT Client", daemon=True
         )
-
         threads.append(mqtt_thread)
 
+        robot_status: RobotStatusPublisher = RobotStatusPublisher(
```

Comment:
Worth adding *_publisher to this and and *Publisher objects below?

Reply:
Done.